### DISCUSSION TITLE
PCIe USB 3 card 'CSL'

### DISCUSSION THREAD

**Discussion by technolhodgy** [geerlingguy/raspberry-pi-pcie-devices#67] [General]
I got this card because it has the same IC as the Pi 4
https://www.amazon.co.uk/gp/product/B00W704W7I/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1

I works straight out of the box just like the Pi4's USB 3 does, giving 5 USB 3 ports.
Although is listed as a 5 Port card, there is also a 15 pin connector which will a cable adds 2 more ports

I bought a second card ( this one is listed as a 7 port card, but look almost identical )
https://www.amazon.co.uk/gp/product/B00Y2F6274/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1
I basically looks like the same card but instead of the internal 15 pin connector there is 2 internal USB 3 ports

My CM4 now has 11 - 4 USB 3 & 7 USB 3
I just need a nice case for it now ( something similar to a Amiga A1200 style would be nice )

@iamhodgy

**Comment by jamesy0ung** [comment]
https://www.waveshare.com/product/raspberry-pi/boards-kits/compute-module-4-cat/pcie-to-usb-3.2-gen1-b.htm
This looks pretty cool and it should solve the issue I have with system lockups when the card is wobbled

**Reply by geerlingguy** [reply]
Woah, that's a nifty little purpose-built card!
